src/rbac/application/common/jwt_token_provider_impl.py

```
@@ -0,0 +1,43 @@
+        return jwt.encode(payload, self.secret_key, algorithm=self.algorithm)
+
+    def __verify_token(self, token: str) -> Dict[str, Any]:
+        try:
+            decoded = jwt.decode(
+                token,
+                self.secret_key,
+                algorithms=[self.algorithm]
+            )
+            return decoded  # dict 형태로 반환됨
+        except PyJWTError as e:
+            raise ValueError(f"Invalid JWT token: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)